app/views/csvfiles/user_files.html.erb

```diff
@@ -0,0 +1,69 @@
+<div class="main_conatiner">
+    <div class="heading_container_data">
+        <h2 class="main_heading_data"> Your Data</h2>
+        <p class="sub_heading_data">View, download and delete CSV files and Research Noted on charts</p>
+    </div>
+</div>
+<table class="table">
+  <thead>
+    <tr>
+      <th scope="col">S.No</th>
+      <th scope="col">File Title</th>
+      <th scope="col">Download</th>
+      <th scope="col">Delete</th>
+      <th scope="col">Description</th>
+    </tr>
+  </thead>
+    <% count=1 %>
```

Comment:
I don't think count is really needed. In the loop, there is already `index` provided which auto increments on every iteration, use it instead of count.

Reply:
So index is not working like an auto incrementer

Reply:
Lets just keep count int there!

Reply:
Using count and then incrementing it everytime isn't a good practice. You need to use `.each.with_index(offset)` if you want to start the index count from a specified number or else use `.each_with_index` to start the index count from 0. This is specified in ruby itself.
There are always multiple ways to solve an issue, but the aim should always be to keep the code simple and effective. A working piece of code doesn't means it is also the correct piece of code for the purpose.